.eslintignore

```diff
@@ -1,3 +1,3 @@
 node_modules
 coverage
-
+dist
```

Comment:
The dist folder is not generated, you only have to change the dist/swagger.yaml
You can use an editor like : http://editor.swagger.io/